src/vmm/src/rpc_interface.rs

```diff
@@ -310,4 +323,150 @@ impl RuntimeApiController {
             )),
         }
     }
+
+    /// Creates a new `RuntimeApiController`.
+    pub fn new(vm_config: VmConfig, vmm: Arc<Mutex<Vmm>>) -> Self {
+        Self { vm_config, vmm }
+    }
+
+    /// Write the metrics on user demand (flush). We use the word `flush` here to highlight the fact
+    /// that the metrics will be written immediately.
+    /// Defer to inner Vmm. We'll move to a variant where the Vmm simply exposes functionality like
+    /// getting the dirty pages, and then we'll have the metrics flushing logic entirely on the outside.
+    fn flush_metrics(&mut self) -> ActionResult {
+        // FIXME: we're losing the bool saying whether metrics were actually written.
+        METRICS
+            .write()
+            .map(|_| ())
+            .map_err(super::Error::Metrics)
+            .map_err(VmmActionError::InternalVmm)
+    }
+
+    /// Injects CTRL+ALT+DEL keystroke combo to the inner Vmm (if present).
+    #[cfg(target_arch = "x86_64")]
+    fn send_ctrl_alt_del(&mut self) -> ActionResult {
+        self.vmm
+            .lock()
+            .unwrap()
+            .send_ctrl_alt_del()
+            .map_err(VmmActionError::InternalVmm)
+    }
+
+    /// Updates the path of the host file backing the emulated block device with id `drive_id`.
+    /// We update the disk image on the device and its virtio configuration.
+    fn update_block_device_path<P: AsRef<Path>>(
+        &mut self,
+        drive_id: &str,
+        disk_image_path: P,
```

Comment:
I would rename `disk_image_path` with `path_on_host` to match with the comment on line 390.

Reply:
done